src/views/CoCurricularTranscript/index.tsx

```diff
@@ -20,7 +28,7 @@ const SectionTitle: { [Key in keyof TranscriptItems]: string } = {
 const CoCurricularTranscript = () => {
   const [loading, setLoading] = useState(true);
   const [transcriptItems, setTranscriptItems] = useState<TranscriptItems | undefined>();
-  const { profile, loading: loadingProfile } = useUser();
+  let { profile, loading: loadingProfile } = useUser();
```

Comment:
Ah interesting. Okay, that makes sense. I am concerned as to what this will do when you switch from the experience transcript to, say, the homepage. Will that still use the example data that you put in here as the profile?

Reply:
If so, you might benefit from using a copy of the profile instead of the actual useUser profile.

Reply:
Yeah, if I understand what you are saying, the data that I wrote would only been used when the users go into the transcript page. If the user switch from the transcript page back to the homepage, for now, they will only see the welcome massage like this:

![image](https://github.com/gordon-cs/gordon-360-ui/assets/78650688/92bc742e-1651-43f5-8299-95bee48a5d91)

Reply:
Great, that answers that!